src/machine.rs

```diff
@@ -534,7 +534,8 @@ impl OlmMachine {
         })
     }
 
-    /// Export all the secrets we have in the store into a [`SecretsBundle`].
+    /// Export all the secrets we have in the store into a {@link
+    /// SecretsBundle}.
```

Comment:
Since this documentation is used by typedoc, we have to use link formats that it understands.

(Note that currently the documentation at https://matrix-org.github.io/matrix-rust-sdk-crypto-wasm/classes/OlmMachine.html#importSecretsBundle doesn't link correctly).